packages/server/src/errorNormalize.ts

```diff
@@ -26,16 +17,25 @@ export function normalizeAndFormatErrors(
       formattedError: GraphQLFormattedError,
       error: unknown,
     ) => GraphQLFormattedError;
-    includeStackTracesInErrorResponses?: boolean;
+    includeStacktraceInErrorResponses?: boolean;
   } = {},
 ): Array<GraphQLFormattedError> {
   const formatError = options.formatError ?? ((error) => error);
   return errors.map((error) => {
     try {
-      return formatError(enrichError(error), error);
+      return formatError(
+        enrichError(error),
+        // If the error was thrown in a resolver, graphql-js will wrap it in a
+        // GraphQLError before it gets back to Apollo Server (to add the path
+        // etc to it). We'd like to give users their actual original error. (If
+        // there is no path, this error didn't come from a resolver.)
+        error instanceof GraphQLError && error.path && error.originalError
+          ? error.originalError
+          : error,
```

Comment:
passing originalError makes sense even if `path` is absent.
example errors thrown from coerceValue/serialize since they also wrapped.
So I recommend dropping the `path` check here.